build/project-template/internal/nsld.sh

```diff
@@ -0,0 +1,53 @@
+ #/bin/bash
+
+ set +o posix
+ set +e
+ source ./env_vars.sh
+ set -e
+
+ MODULES_DIR=$SRCROOT/internal/Swift-Modules
+
+ function DELETE_SWIFT_MODULES_DIR() {
+  rm -rf $MODULES_DIR
+}
+
+function GEN_MODULEMAP() {
+  DERIVED_SOURCES=$DERIVED_FILES_DIR
+
+  if [ -d "$DERIVED_SOURCES" ]; then
+    count=`ls $DERIVED_SOURCES/*-Swift.h 2>/dev/null | wc -l`
+
+      if [ $count == 1 ]
+      then
+          HEADER_PATH=`find $DERIVED_SOURCES -name *-Swift.h`
+          DELETE_SWIFT_MODULES_DIR
+
+          mkdir -p $MODULES_DIR
+          CONTENT="module nsswiftsupport { \n header \"$HEADER_PATH\" \n export * \n}"
+          printf "$CONTENT" > "$MODULES_DIR/module.modulemap"
+      else
+      echo "-Swift.h not found!"
```

Comment:
This output will be visible in the whole build log, we should make it easier to understand its origins and meaning. I'd suggest something like: `NSLD: Swift bridging header "*-Swift.h" not found`